### PR TITLE
fix topInstructions eyes test

### PR DESCRIPTION
**Pull request by Hamms**
By updating the test itself to pay attention to the new inline feedback rather than the old dialog feedback.

Also cleans up some React warnings and removes the logic preventing shortInstructions from "uncollapsing".

### CODE REVIEW

**Comment by Hamms** [comment]
FYI @ashercodeorg fix for the eyes test. Note that this will still result in an eyes change, which is to be expected. I'm happy to help review the changes when the time comes.

File: apps/src/templates/instructions/InlineFeedback.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import Radium from 'radium';
 
 const InlineFeedback = ({ extra, message, styles }) => {
   return (
-    <div style={styles.container}>
+    <div style={styles.container} className="topInstructions-inline-feedback">
```
Comment (Bjvanminnen) [inline apps/src/templates/instructions/InlineFeedback.jsx]: I wonder if we can make it more clear that this is only used by UI tests? Some options I see
(1) Just add a comment
(2) change the name (something like `uitest-topInstructions-inline-feedback`)
(3) Use a data attribute instead, i.e. `data-inline-feedback={true}`

Thoughts?

The goal would be to make it clear that one doesnt need to worry about styles being applied via this classname.

**Comment by Bjvanminnen** [comment]
lgtm

File: apps/src/templates/instructions/InlineFeedback.jsx (hunk 1 above)
Comment (Hamms) [inline apps/src/templates/instructions/InlineFeedback.jsx]: Hmm. I think I like 1+2.

**Pull request merged**